src/events/ClientReady.ts

```diff
@@ -1,6 +1,5 @@
 import type GoStudyCommand from "@/commands/miscellaneous/GoStudy";
 import type HostSessionCommand from "@/commands/study/HostSession";
-import type PracticeCommand from "@/commands/study/Practice";
```

Comment:
This will cause Practice Sessions to stop working and is unrelated to the PR

Reply:
This should be fixed in https://github.com/Sachin-dot-py/r-igcse-bot/pull/129/commits/63437a638a74a1f527e421da9532f9ddf306d29a